app/controllers/loans_controller.rb

```diff
@@ -0,0 +1,33 @@
+class LoansController <  ApplicationController
+  before_action :set_loan, only: [:show]
+  def new
```

Comment:
I strongly suggest the use of the gem `Rubocop`, in that way you can realize if some code style in your code is wrong. When you are coding alone it's not that important, but when working in group your code needs to be standarized and also pleased the others when reading.
For example here a space between the before action and the method is needed 😊 
```
before_action :set_loan, only: [:show]

def new
...
```